packages/graph-editor/src/editor/actions/duplicate.ts

```diff
@@ -31,14 +31,22 @@ export const duplicateNodes = ({ graph, reactFlowInstance, nodeLookup }: IDuplic
             }
             const newID = uuidv4();
             const saved = graphNode.serialize();
+            const newPosition = {
+                x: node.position.x + 20,
+                y: node.position.y + 100,
+            }
 
             const newGraphNode = graphNode?.factory.deserialize({
                 serialized: {
                     ...saved,
                     id: newID,
+                    annotations: {
```

Comment:
We should probably spread the other annotations here so that they are not lost

Reply:
done